### PR TITLE
star: readme file

### PR DESCRIPTION
**Pull request by jemgillam**
## Description

A readme file for the `utils` folder - which we colloquially call the "star repo" 

Note the updated shields badges. I have removed Patreon and Twitter, and added Mastodon and Bluesky. I have used the official GitHub sponsors pink. 

Screenshot 2024-12-10 at 12 23 07
Screenshot 2024-12-10 at 12 23 44

If you like that change I can do this across our projects. we should at least remove Patron from them anyway. 

This readme includes the sponsors block, so I should hopefully be able to update that block automatically when I run the sponsors update.

### CODE REVIEW

File: utils/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Graphile\*
+
+[![GitHub Sponsors](https://img.shields.io/github/sponsors/benjie?color=%23ea4aaa&label=github%20sponsors)](https://github.com/sponsors/benjie)
+[![Discord chat room](https://img.shields.io/discord/489127045289476126.svg)](http://discord.gg/graphile)
+[![Mastodon Follow](https://img.shields.io/mastodon/follow/109982006777922021?domain=https%3A%2F%2Ffosstodon.org&style=flat&label=mastodon&color=%236364FF)](https://fosstodon.org/@graphile)
+[![Static Badge](https://img.shields.io/badge/bsky-graphile.org-blue?style=flat&color=%230285FF)](https://bsky.app/profile/graphile.org)
```
Comment (benjie) [inline utils/README.md:6]:
```suggestion
[![Bluesky](https://img.shields.io/badge/bsky-graphile.org-blue?style=flat&color=%230285FF)](https://bsky.app/profile/graphile.org)
```

File: utils/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+_**Graphile\* - the stars we forge to build our universe**_
+
+**Documentation**: https://star.graphile.org
+
+## About
+
+A collection of useful utilities which work in tandem with the rest of the
+Graphile suite of open source developer tools.
```
Comment (benjie) [inline utils/README.md:15]:
```suggestion
A collection of useful utilities we built whilst working on Graphile suite of
open source developer tools, that may well be useful for your projects too.
```

**Review by benjie — CHANGES_REQUESTED** [review]

**Review by benjie — DISMISSED** [review]
Looks good to me! Go ahead and merge if you're happy with it.

File: utils/README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+[![Mastodon Follow](https://img.shields.io/mastodon/follow/109982006777922021?domain=https%3A%2F%2Ffosstodon.org&style=flat&label=mastodon&color=%236364FF)](https://fosstodon.org/@graphile)
+[![Bluesky](https://img.shields.io/badge/bsky-graphile.org-blue?style=flat&color=%230285FF)](https://bsky.app/profile/graphile.org)
+
+_**Graphile\* - the stars we forge to build our universe**_
+
+**Documentation**: https://star.graphile.org
+
+## About
+
+A collection of useful utilities we built whilst working on Graphile suite of
```
Comment (jemgillam) [inline utils/README.md:14]:
```suggestion
A collection of useful utilities we built whilst working on the Graphile suite of
```

**Pull request merged**